docs/content/releases.md

```diff
@@ -7,6 +7,36 @@ toc: true
 docs: "DOCS-616"
 ---
 
+## NGINX Ingress Controller 2.2.0
+
+12 Apr 2022
+
+FEATURES:
+* [2576](https://github.com/nginxinc/kubernetes-ingress/pull/2576) Add support for IPv6.
+* [2572](https://github.com/nginxinc/kubernetes-ingress/pull/2572) Automate provisioning of Certificate resources for VirtualServer resources using cert-manager.
+
+ENHANCEMENTS:
+* [2346](https://github.com/nginxinc/kubernetes-ingress/pull/2346) Use os.ReadDir for lightweight directory reading. Thanks to [Eng Zer Jun](https://github.com/Juneezee).
+* [2360](https://github.com/nginxinc/kubernetes-ingress/pull/2360) Add NGINX App Protect reconnect period directive.
+* [2479](https://github.com/nginxinc/kubernetes-ingress/pull/2479) Add cli argument to configure NGINX App Protect log level.
+* [2455](https://github.com/nginxinc/kubernetes-ingress/pull/2455) Increase memory available for NGINX App Protect xml parser.
+
+CHANGES:
+* [2580](https://github.com/nginxinc/kubernetes-ingress/pull/2580) Create -enable-oidc command line argument for OIDC policy.
+* [2566](https://github.com/nginxinc/kubernetes-ingress/pull/2566) Unbind policy from preview policies.
+
+FIXES:
+* [2378](https://github.com/nginxinc/kubernetes-ingress/pull/2378) Fix healthcheck ports.
+* [2404](https://github.com/nginxinc/kubernetes-ingress/pull/2404) Start nginx with -e stderr parameter.
+* [2414](https://github.com/nginxinc/kubernetes-ingress/pull/2414) Fix in file nginx-plus.virtualserver.tmpl ApDosMonitor->ApDosMonitorURI.
+
+HELM CHART:
+* [2525](https://github.com/nginxinc/kubernetes-ingress/pull/2525) Extend helm chart to include NGINX Service Mesh fields.
+* [2294](https://github.com/nginxinc/kubernetes-ingress/pull/2294) Add extra containers to helm chart. Thanks to [Márk Sági-Kazár](https://github.com/sagikazarmark).
+
+
+For a complete list of changes, upgrade instructions and resources, see the [NGINX Ingress Controller Release Notes](https://github.com/nginxinc/kubernetes-ingress/releases/tag/v2.0.0).
```

Comment:
We'll need a details for upgrade instructions here too (RBAC needs to be updated, details on the deprecated flag, and new `-enable-oidc` flag)

Reply:
```suggestion
UPGRADE:
* For NGINX, use the 2.2.0 images from our [DockerHub](https://hub.docker.com/r/nginx/nginx-ingress/tags?page=1&ordering=last_updated&name=2.1.0), [GitHub Container](https://github.com/nginxinc/kubernetes-ingress/pkgs/container/kubernetes-ingress) or [Amazon ECR Public Gallery](https://gallery.ecr.aws/nginx/nginx-ingress).
* For NGINX Plus, use the 2.2.0 images from the F5 Container registry or build your own image using the 2.2.0 source code.
* For Helm, use version 0.13.0 of the chart. If you're using custom resources like VirtualServer and TransportServer (`controller.enableCustomResources` is set to `true`), after you run the `helm upgrade` command, the CRDs will not be upgraded. After running the `helm upgrade` command, run `kubectl apply -f deployments/helm-chart/crds` to upgrade the CRDs.
* When upgrading using the [manifests](https://docs.nginx.com/nginx-ingress-controller/installation/installation-with-manifests/), make sure to update the [ClusterRole](https://github.com/nginxinc/kubernetes-ingress/blob/v2.2.0/deployments/rbac/rbac.yaml). This is required to enable the cert-manager for VirtualServer resources integration.
* The -enable-preview-policies cli argument has been deprecated, and is no longer required for any Policy resources.
* Enabling OIDC Policies now requires the use of -enable-oidc cli argument instead of the -enable-preview-policies cli argument.
SUPPORTED PLATFORMS:
We will provide technical support for the NGINX Ingress Controller on any Kubernetes platform that is currently supported by its provider and which passes the Kubernetes conformance tests. This release was fully tested on the following Kubernetes versions: 1.19-1.23.
```

Reply:
Do we really need to add the first 3 points? I thought about adding the link to the release notes on GitHub since the upgrade notes are there too (and automatically generated)

Reply:
Yeah it seemed a little redundant and I decided to remove it (after I was done writing it) 😄

Reply:
Yep! I think the additional upgrade and supported platforms make sense here.

Reply:
Well I've added it all, we can think about it for the next release